src/components/DynamicCalendar/manifest.json

```diff
@@ -9,7 +9,8 @@
     {
       "name": "items",
       "displayName": "What events should be displayed?",
-      "type": "list"
+      "type": "list",
```

Comment:
In the future, make sure to only commit the changes for this PR. But in this case it'll be easy to just close the other PR and merge this one in.